packages/transactions/src/keys.ts

```diff
@@ -31,6 +37,12 @@ import {
 import { BufferReader } from './bufferReader';
 import { c32address } from 'c32check';
 
+utils.hmacSha256Sync = (key: Uint8Array, ...msgs: Uint8Array[]) => {
+  const h = hmac.create(sha256, key);
+  msgs.forEach(msg => h.update(msg));
+  return h.digest();
+};
```

Comment:
Why is a lib's function being defined here? Can you add a comment?

Reply:
Done!

Reply:
Thanks!